news/3907.bugfix

```diff
@@ -0,0 +1 @@
+Abort the installation process and raise an exception if one of the tar/zip file will be placed outside of target location causing security issue.
```

Comment:
I would say "Abort installation if any archive contains a file which would be placed outside the extraction location."

Reply:
I second this suggestion.